### PR TITLE
Add `eslint-plugin-github` and fix issues

### PR DESCRIPTION
**Pull request by silverwind**
This plugin has a few useful rules. The only thing I dislike about it is that it pulls in a rather big number of dependencies for react-related rules we don't use, but it can't really be avoided.

Rule docs: https://github.com/github/eslint-plugin-github?tab=readme-ov-file#rules

### CODE REVIEW

File: web_src/js/features/repo-issue-list.js [hunk 1]
```diff
@@ -69,16 +69,14 @@ function initRepoIssueListCheckboxes() {
...
+    try {
+      await updateIssuesMeta(url, action, issueIDs, elementId);
       window.location.reload();
-    }).catch((reason) => {
-      showErrorToast(reason.responseJSON.error);
-    });
+    } catch (err) {
+      if (err?.responseJSON?.error) {
+        showErrorToast(err.responseJSON.error);
+      }
```
Comment (wxiaoguang) [inline web_src/js/features/repo-issue-list.js]: Add a "else" ? Ignoring errors silently might hide some bugs.

Comment (silverwind) [reply web_src/js/features/repo-issue-list.js]: Good point. Even thought `window.location.reload` is unlikely to throw, we can add it.

**Review by wxiaoguang — APPROVED** [review]

File: web_src/js/features/repo-issue-list.js [hunk 2]
```diff
@@ -69,16 +69,12 @@ function initRepoIssueListCheckboxes() {
...
-      elementId,
-    ).then(() => {
+    try {
+      await updateIssuesMeta(url, action, issueIDs, elementId);
       window.location.reload();
-    }).catch((reason) => {
-      showErrorToast(reason.responseJSON.error);
-    });
+    } catch (err) {
+      showErrorToast(err?.responseJSON?.error ?? err.message);
```
Comment (delvh) [inline web_src/js/features/repo-issue-list.js]: First, `err` can be null, but then in the else block it suddenly can't be anymore?

**Review by delvh — APPROVED** [review]

File: web_src/js/features/repo-issue-list.js (hunk 2 above)
Comment (silverwind) [reply web_src/js/features/repo-issue-list.js]: `err` can't be null, right.

Comment (silverwind) [inline web_src/js/features/repo-issue-list.js]:
```suggestion
      showErrorToast(err.responseJSON?.error ?? err.message);
```

**Pull request merged**